### ISSUE TITLE
"uncaught exception: undefined" upon joining a group

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#703]
### Problem

A mysterious exception appears in the console upon joining a group:

```
uncaught exception: undefined
```

Surrounding console output:

```
[Backend] subscribed 21XWnNFQpc3m3v7EiyTdandFuUHPdVEbXVUoJ8vz9mqjeB2K7g 
Object { type: "success", data: {…} }
backend.js:87:14
[addLogEntry] HEAD for 21XWnNFQpc3m3v7EiyTdandFuUHPdVEbXVUoJ8vz9mqjeB2K7g updated to: 21XWnNFxmAu9KgvPD6foWHCn1PunmCJQeev8RVfPfV2Ce2mZiw db.js:66:14
[sbp] gi.contracts/identity/setAttributes/process 
Array [ {…}, {…} ]
main.js:41:14
[sbp] okTurtles.events/emit 
Array(3) [ "21XWnNFxmAu9KgvPD6foWHCn1PunmCJQeev8RVfPfV2Ce2mZiw", "21XWnNFQpc3m3v7EiyTdandFuUHPdVEbXVUoJ8vz9mqjeB2K7g", {…} ]
main.js:41:14
[sbp] okTurtles.events/emit 
Array(3) [ "event-handled", "21XWnNFQpc3m3v7EiyTdandFuUHPdVEbXVUoJ8vz9mqjeB2K7g", {…} ]
main.js:41:14
[sbp] okTurtles.events/emit 
Array(3) [ "21XWnNFjyvfzLdwQkZnRkB81iUoSq9Zs9odXgH6HzkpVDo1M2D", "21XWnNQkTCqEY49ppuQvVSqMALN3t7CSVNQiR2tTfxfSVcpSsD", {…} ]
main.js:41:14
[sbp] okTurtles.events/emit 
Array(3) [ "event-handled", "21XWnNQkTCqEY49ppuQvVSqMALN3t7CSVNQiR2tTfxfSVcpSsD", {…} ]
main.js:41:14
Avatar under vue-component-33-UserImage blobURL: undefined src: http://localhost:8000/assets/images/default-avatar.png 2 Avatar.vue:25:12
uncaught exception: undefined
[sbp] gi.db/settings/save 
Array [ "bob", {…} ]
```

### Solution

Add to `main.js` a handler for uncaught exceptions (e.g. [`Vue.config.errorHandler`](https://vuejs.org/v2/api/#errorHandler) or [`window.onerror`](https://blog.sentry.io/2016/01/04/client-javascript-reporting-window-onerror), whichever one is better -- justify your decision). Use it to print the stack trace (`err.stack`) and find out where this error is coming from, and fix it.

See also related #684

**Comment by taoeffect** [comment]
NOTE: I tried closing this issue but neither `Vue.config.errorHandler` nor `window.onerror` were able to show where the error is coming from... :-\

**Comment by sandrina-p** [comment]
I tried to replicate this error but it didn't happen (chrome and firefox). I believe I fixed this at #747.

Closing this. If you still see the error, please specify better the steps to reproduce it and feel free to reopen the issue.
